app/controllers/users/emails_controller.rb

```diff
@@ -1,9 +1,12 @@
 module Users
-  class EmailsController < ReauthnRequiredController
```

Comment:
I don't see the controller implementation removed from code. Should it be?

Reply:
whoops, yep, thank you